pkg/ui/workspaces/cluster-ui/src/api/clusterSettingsApi.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { cockroach } from "@cockroachlabs/crdb-protobuf-client";
+import { fetchData } from "src/api";
+import { ADMIN_API_PREFIX } from "./util";
+
+export type SettingsRequestMessage = cockroach.server.serverpb.SettingsRequest;
+export type SettingsResponseMessage =
+  cockroach.server.serverpb.SettingsResponse;
+
+// getClusterSettings gets all cluster settings. We request unredacted_values, which will attempt
+// to obtain all values from the server. The server will only accept to do so if
+// the user also happens to have admin privilege.
```

Comment:
Is there a mention of adding this api in the commit? I might also have missed where this came from but it might be a good idea to summarize all the changes required to get the connected component up for CC, including these cluster setting changes.

Reply:
Updated.